dataprofiler/profilers/profiler_options.py

```diff
@@ -982,6 +982,64 @@ def _validate_helper(self, variable_path: str = "CorrelationOptions") -> list[st
         return errors
 
 
+class HyperLogLogOptions(BooleanOption):
```

Comment:
Created a nested HLL options class because seed is only pertinent when HLL is enabled.

Reply:
Let's rename to `UniqueCountOptions`. Here we should have a str choice to use `full` or `hll_row_hashing` in addition we can have the hll_options.